src/components/pages/create-account/EligibilityCheck/EligibilityCheck.jsx

```diff
@@ -1,9 +1,70 @@
-import React from 'react'
+import React, { useState } from 'react'
+import InputMask from 'react-input-mask'
 
-const EligibilityCheck = () => (
-  <main>
-    {' EligibilityCheck '}
-  </main>
-)
+import BackLink from '../../../layout/Header/BackLink/BackLink'
+
+const EligibilityCheck = () => {
+  let [postalCodeInputValue, setPostalCodeInputValue] = useState('')
+  let [dateOfBirthInputValue, setDateOfBirthInputValue] = useState('')
+
+  let handlePostalCodeInputChange = event => {
```

Comment:
```suggestion
  const handlePostalCodeInputChange = event => {
```
Changer tous les autres.